Parse/src/test/java/com/parse/ParseClientConfigurationTest.java

```diff
@@ -30,14 +30,14 @@ public void testBuilder() {
     Parse.Configuration.Builder builder = new Parse.Configuration.Builder(null);
     builder.applicationId("foo");
     builder.clientKey("bar");
-    builder.server("some.server");
+    builder.server("http://myserver.com/parse");
```

Comment:
Can we perhaps duplicate this test case, instead of just modifying it in-place? One test for each branch?